src/guides/libraries/useful-libraries.md

```diff
@@ -16,38 +16,39 @@ Commonly used packages fall into three groups:
 
 ## General-purpose packages
 
-The following packages are useful for a wide range of projects.
+Use these packages for a wide range of projects.
 
 | **Package** | **Description** | **Commonly used APIs** |
+|---------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | [archive]({{site.pub-pkg}}/archive) | Encodes and decodes various archive and compression formats. | Archive, ArchiveFile, TarEncoder, TarDecoder, ZipEncoder, ZipDecoder |
-| [characters]({{site.pub-pkg}}/characters) | String manipulation for user-perceived characters (Unicode grapheme clusters). | String.characters, Characters, CharacterRange |
-| [http]({{site.pub-pkg}}/http) | A set of high-level functions and classes that make it easy to consume HTTP resources. | delete(), get(), post(), read() |
+| [characters]({{site.pub-pkg}}/characters) | Manipulates strings for user-perceived characters (Unicode grapheme clusters). | String.characters, Characters, CharacterRange |
+| [cronet_http]({{site.pub-pkg}}/cronet_http) | Provides access to the Android [Cronet](https://developer.android.com/guide/topics/connectivity/cronet/reference/org/chromium/net/package-summary) HTTP client. | |
```

Comment:
for this an `cupertino_http`, adding the words "using the same interface as `package:http`" might make it more clear how they fit in.